packages/esm-patient-lists-app/src/workspaces/patient-list-details-table.component.tsx

```diff
@@ -35,6 +35,10 @@ const PatientListDetailsTable: React.FC<PatientListDetailsTableProps> = ({ listM
   const [searchTerm, setSearchTerm] = useState('');
   const debouncedSearchTerm = useDebounce(searchTerm);
 
+  if (!listMembers) {
+    return null; // or return a loading indicator or placeholder
```

Comment:
An `isLoading` condition below renders a loading skeleton. Error states are handled using the `ErrorState` component.